src/magic.h

```diff
@@ -297,6 +308,11 @@ class known_magic
 
         void serialize( JsonOut &json ) const;
         void deserialize( JsonIn &jsin );
+    private:
+        // gets length of longest spell name
+        size_t get_spellname_max_width();
+        // gets invlet if assigned, or -1 if not
+        int get_invlet( spell_id &sp, std::set<int> &used_invlets );
```

Comment:
Are you sure you don't want to use const here?

```suggestion
        int get_invlet( const spell_id &sp, std::set<int> &used_invlets );
```